### DISCUSSION TITLE
Request for Hungarian language on poeditor.com

### DISCUSSION THREAD

**Discussion by v1k70rk4** [Koenkk/zigbee2mqtt#18692] [General]
Hi, I'm not sure if I'm reporting this in the right place, but if you put Hungarian on poeditor.com, I'd be happy to translate it :)

**Comment by Koenkk** [comment]
Requests related to the frontend should be created here: https://github.com/nurikk/zigbee2mqtt-frontend/issues

**Comment by v1k70rk4** [comment]
Thank you very much for the help :)
